packages/swapper/src/swappers/baseErrors.ts

```diff
@@ -0,0 +1,10 @@
+export enum SWAPPER_ERRORS {
+  SLIPPAGE_EXCEEDED = 'trade.errors.slippageExceeded',
+  UNDER_MINIMUM_AMOUNT = 'trade.errors.underMinimumAmount',
+  CONSUMED_MORE_FEES = 'trade.errors.consumedMoreFees',
```

Comment:
It can be weird to have translation keys in the swapper package, but for the sake of simplicity, I wanted to keep it here

The very clean way would be to return error codes or strings, and map these errors codes/strings to our translation keys on react app side, but the swapper is not consumed outside of this scope

Depending on reviews args, we might want to do it, but my choice has been done already 😄

Reply:
For the sake of simplicity, would actually prefer to keep just indexes. This is redeclaring keys from `JUPITER_ERROR_VALUES`, reading this is a big indirection for what should be a simple mapping (three count between `JUPITER_ERROR_VALUES`, redundant `JUPITER_ERRORS`,  and this